python/server/ServerWrapper.cpp

```diff
@@ -116,7 +119,7 @@ namespace {
         } else {
             Empire* empire = GetEmpire(empire_id);
             if (!empire) {
-                ErrorLogger() << "PythonServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
+                ErrorLogger() << "ServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
```

Comment:
ErrorLogger is an alias for BOOST_LOG_TRIVIAL(error) which outputs whatever is streamed into it to the appropriate log file using Boost.Log. It's probably possible but probably complicated to get it to output the file and line number in that file from where the log output was generated, if that's what you meant...